stdlib/public/core/String.swift

```diff
@@ -409,11 +409,15 @@ extension String {
   @inline(never) // slow-path
   private static func _fromNonContiguousUnsafeBitcastUTF8Repairing<
     C: Collection
-  >(_ input: C) -> (result: String, repairsMade: Bool) {
+  >(_ input: C) -> String {
     _internalInvariant(C.Element.self == UInt8.self)
-    return Array(input).withUnsafeBufferPointer {
-      let raw = UnsafeRawBufferPointer($0)
-      return String._fromUTF8Repairing(raw.bindMemory(to: UInt8.self))
```

Comment:
The call to `_fromUTF8Repairing` mattered here, as it validates the input.

Reply:
Thank you for reviewing it while knowing that this won't land! ❤️
I'll take a look at @karwa's PR and learn how he managed to do this.

Reply:
Oh, I see that the validation result was being discarded anyway.